packages/core/utils/lib/convert-query-params.js

```diff
@@ -218,7 +218,77 @@ const convertFieldsQueryParams = (fields, depth = 0) => {
   throw new Error('Invalid fields parameter. Expected a string or an array of strings');
 };
 
-const convertFiltersQueryParams = filters => filters;
+const convertFiltersQueryParams = (filters, schema) => {
+  // Filters need to be either an array or an object
+  // Here we're only checking for 'object' type since typeof [] => object and typeof {} => object
+  if (!isObject(filters)) {
+    throw new Error('The filters parameter must be an object or an array');
+  }
+
+  const sanitizeFilters = (filters, schema) => {
+    if (!isObject(filters)) {
+      return filters;
+    }
+
+    if (Array.isArray(filters)) {
+      return (
+        filters
+          // Sanitize each filter
+          .map(filter => sanitizeFilters(filter, schema))
+          // Filter out empty filters
+          .filter(filter => !isObject(filter) || !isEmpty(filter))
+      );
+    }
+
+    // Here, `key` can either be an operator or an attribute name
+    for (const [key, value] of Object.entries(filters)) {
+      const removeOperator = () => delete filters[key];
```

Comment:
Should we avoid to mutate the filters? The other functions `convert` seem to not mutate the args